lib/agent.js

```diff
@@ -433,3 +433,7 @@ Agent.prototype.flush = function (cb) {
 function normalizeTimestamp (timestamp) {
   return (timestamp > 0 && Math.floor(timestamp * 1000)) || Date.now() * 1000
 }
+
+Agent.prototype.registerCustomMetric = function (name, callback, labels) {
```

Comment:
How about we just call this `registerMetric`? I feel like having "custom" in the name makes them feel second class.

Reply:
Sounds good.